extractor/tests/test_extractor.py

```diff
@@ -6,11 +6,13 @@
 import tempfile
 import shutil
 
+DOCKER_REGISTRY = os.environ.get('DOCKER_REGISTRY', '')
+
 
 class TestExtractorE2E(TestCase):
     """Ensure that the extractor can be built, and performs as expected."""
 
-    def test_extract_fulltext(self):
+    def test_extract(self):
```

Comment:
It is excellent that this gets tested. It would be great to include more pdfs in this test.